packages/query/src/graphql/plugins/GetMetadataPlugin.ts

```diff
@@ -3,10 +3,27 @@
 
 import {makeExtendSchemaPlugin, gql} from 'graphile-utils';
 import fetch from 'node-fetch';
+import {setAsyncInterval} from '../../utils/asyncInterval';
 import {argv} from '../../yargs';
 
 const metaUrl = argv('meta') as string | undefined;
 
+type MetaData = {
```

Comment:
I think we only keep the following: `targetHeight`, `lastProcessedHeight`, `lastProcessedTimestamp`, `chain`, `specName`, `genesisHash`, `indexerHealthy`, `indexerNodeVersion`, `queryNodeVersion`